recipe/meta.yaml

```diff
@@ -167,7 +167,6 @@ outputs:
         - ripgrep
         - cmake
         - ninja
-        - cython
```

Comment:
Looks like that was not it, or not just it.

What should we do about https://github.com/conda-forge/python-feedstock/pull/389/files#diff-e178b687b10a71a3348107ae3154e44cR124?